stash_engine/spec/db/stash_engine/resource_spec.rb

```diff
@@ -261,29 +261,33 @@ module StashEngine
     end
 
     describe :public? do
+
+      before(:each) do
+        @resource = Resource.create(user_id: user.id)
+        allow(@resource).to receive(:submit_to_solr).and_return(true)
+      end
+
       it 'defaults to true' do
-        resource = Resource.create(user_id: user.id)
-        expect(resource.public?).to eq(true)
+        expect(@resource.public?).to eq(true)
       end
 
       it 'returns true for expired embargoes' do
-        resource = Resource.create(user_id: user.id)
-        StashEngine::Embargo.create(resource_id: resource.id, end_date: Date.new(2015, 5, 18))
-        resource.reload
-        expect(resource.public?).to eq(true)
+        StashEngine::Embargo.create(resource_id: @resource.id, end_date: Date.new(2015, 5, 18))
```

Comment:
Is this `StashEngine::Embargo` still relevant? I'm using the curation activity to mark 'embargoed' and the `resources.publication_date` for the date the embargo ends.

Reply:
Nope, it's not relevant then if you're not using it.  We probably need to clean up the old Embargo object, delete tests and eventually drop it from the database.  I suppose we also need a way to update the new embargo method from the old if any Dash records still have an embargo when we release.

It may be just as easy to update by hand because we probably only have a handful of embargoed datasets in Dash where the embargo hasn't already passed.